### ISSUE TITLE
Misaligned String Placeholders leading to Incorrect gcloud Commands

### ISSUE THREAD

**Issue by hinewin** [pfnet-research/gcp-workload-identity-federation-webhook#25]
I've encountered an issue where the gcloud command generated by the code doesn't execute as intended. The `credential-source-file` and  `output-file` values get swapped, due to misaligned string placeholders.

Here is the piece of code in question:
```
  gcloud iam workload-identity-pools create-cred-config \
      $(GCP_WORKLOAD_IDENTITY_PROVIDER) \
      --service-account=$(GCP_SERVICE_ACCOUNT) \
      --output-file=$(CLOUDSDK_CONFIG)/%s \
      --credential-source-file=%s
      gcloud auth login --cred-file=$(CLOUDSDK_CONFIG)/%s
`, filepath.Join(K8sSATokenMountPath, K8sSATokenName),
  ExternalCredConfigFilename,
  ExternalCredConfigFilename,
```

**Comment by sosimon** [comment]
Link to relevant code: https://github.com/pfnet-research/gcp-workload-identity-federation-webhook/compare/v0.3.2...v0.4.0#diff-7950d00968cf05b57d3dc0dc73c56d3abe3e2d58528d20e0c19a1ee44ea42b69L79-R119

**Comment by ordovicia** [comment]
Thanks for opening an issue! I will take a look.

**Comment by ordovicia** [comment]
Newly released v0.4.1 includes a fix for this bug. Thank you for reporting!
